piksi_multi_rtk_ros/launch/geodetic_survey.launch

```diff
@@ -4,7 +4,8 @@
 
   <!-- Parameters. -->
   <arg name="number_of_desired_fixes"                 default="5000"/>
-  <arg name="height_base_station_from_ground"         default="0.0"/>
+  <!-- Height antenna base station from ground for RSL-ETHZ tripod -->
+  <arg name="height_base_station_from_ground"         default="1.89"/>
```

Comment:
While I like this personally, this might introduce unwanted/unexpected issues to other users, as they do not know about the base station antenna pole we are using. Should we consider to have a flag for this or use a different parameter that overwrites this one?

Reply:
Umh, this parameter is used only to compute the origin of the ENU frame that is saved on a file. So the surveyed position (and the parameters set in Piksi) are not affected by this parameter. I'd just keep this param here and add a comment about its usage ...

Reply:
Yes, makes sense!